config/initializers/devise.rb

```diff
@@ -0,0 +1,285 @@
+require 'JsonFailureApp'
+# frozen_string_literal: true
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
```

Comment:
oh boy lol, I love devise but the comments in the code are really distracting when it comes to reviewing haha.